src/sqlfluff/dialects/dialect_postgres_keywords.py

```diff
@@ -446,6 +448,7 @@ def get_keywords(keyword_list, keyword_type):
     ("LOGGED", "non-reserved"),
     ("LOWER", "not-keyword"),
     ("M", "not-keyword"),
+    ("MAIN", "non-reserved"),
```

Comment:
This list is made to match the official docs (https://www.postgresql.org/docs/14/sql-keywords-appendix.html). Could you move this keyword down to the `postgres_nondocs_keywords` section below?